packages/ui/src/views/repo/pull-request/pull-request.types.ts

```diff
@@ -49,10 +49,34 @@ export interface PullRequestListStore {
   closedPullReqs: number
 }
 export interface RepoRepositoryOutput {
+  created?: number
```

Comment:
This interface can be simplified by removing all the `?`s and `null`s. These fields can always receive a default value (empty string or `0` from the consumer layer which is in `gitness`. I understand that these need to be similar to the actual interfaces from Code service client but don't have to be exactly same.

Reply:
will take this up when i refactor it as mentioned in standup